plutus-tx/src/Language/PlutusTx/Prelude.hs

```diff
@@ -212,6 +213,9 @@ all = [||
         in go l
     ||]
 
+txhash :: Q (TExp ByteString)
+txhash = [|| Builtins.txhash ||]
```

Comment:
Added this one, because got errors related to `TxHash`, but apparently they seem to be unrelated.